openquake/commonlib/tests/calculators/__init__.py

```diff
@@ -63,14 +63,14 @@ def execute(self, testfile, job_ini):
         self.calc.pre_execute()
         return self.calc.execute()
 
-    def assertEqualFiles(self, fname1, fname2):
+    def assertEqualFiles(self, fname1, fname2, wraplines=lambda line: line):
```

Comment:
Changed to  "make_comparable" and added a long docstring explaining the use case